dashboard/test/lib/pd/payment/payment_calculator_csf_test.rb

```diff
@@ -67,5 +67,22 @@ class PaymentCalculatorCSFTest < ActiveSupport::TestCase
 
       assert_equal 50, payment.total
     end
+
+    # Any CSF teacher in the workshop section with a valid enrollment counts as attended.
+    # We had an issue where teachers with enrollments from another workshop were in this workshop's
+    # section for some reason, and it caused errors. Verify now they are excluded and cause no errors.
+    test 'teachers in section with enrollments in another workshop are not counted' do
+      external_teacher = create :teacher
+      create :pd_enrollment, user: external_teacher
+      10.times do
+        create :user_level, user: external_teacher, best_result: ::ActivityConstants::MINIMUM_PASS_RESULT
```

Comment:
Dumb Q: Is there a reason to prefer `::ActivityConstants::MINIMUM_PASS_RESULT` over `ActivityConstants::MINIMUM_PASS_RESULT`? Don't know myself.